pages/community/precice-workshop-2022.md

```diff
@@ -19,12 +19,203 @@ If you have any questions regarding the workshop, please contact us at [`precice
 
 ## Program
 
-The preliminary program reads as follows (more details will follow soon):
+The workshop stretches from Monday noon to Thursday evening.
```

Comment:
Lets mention the timezone here and also all the other places where we mention time. So write CET